Patient Record Visualization

radamson opened the pull request:
Adds https://github.com/synthetichealth/fhir-visualizers and displays the patient record.  As a "template" app I'm sure these will be used differently, but usage is exemplified in `App.js`.

What the example looks like: 
![Screen Shot 2019-11-04 at 3 41 54 PM](https://user-images.githubusercontent.com/41651655/68158921-b4447d80-ff1e-11e9-9abb-02dfbac84057.png)

dehall commented:
This looks good. The styling on the visualizers could be improved (for the record, I created the initial CSS; I'm not ragging on Reece) but making it look perfect is not worth spending a ton of time on at the moment.
@radamson  Can you also create a PR on the fhir-visualizers project to get the package branch referenced here merged into master?

dehall commented on package.json:
```diff
@@ -7,6 +7,7 @@
     "@types/node": "^12.11.1",
     "@types/react": "^16.9.9",
     "@types/react-dom": "^16.9.2",
+    "fhir-visualizers": "https://github.com/synthetichealth/fhir-visualizers#package",
```
I merged the PR on fhir-visualizers, so this can be updated. Other than this it looks good, but I'd like to wait until after Noranda's upcoming PR.
```suggestion
    "fhir-visualizers": "synthetichealth/fhir-visualizers",
```

radamson commented:
Rebased to latest master.  Broke the patient record into its own component to avoid polluting `App.tsx`.  Added a super simple declaration file for the `fhir-visualization` library to aid in Typescript usage.
